setup.py

```diff
@@ -7,6 +7,7 @@
 setup(
     name='contextlib2',
     version=open('VERSION.txt').read().strip(),
+    python_requires='>=2.7',
```

Comment:
This would also allow installation on Python 3.0-3.3. To exclude those:

```suggestion
    python_requires='>=2.7, !=3.0.*, !=3.1.*, !=3.2.*, !=3.3.*',
```

Reply:
Yep you right. And surprisingly fast.